pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.50</version>
+    <version>4.66</version>
```

Comment:
I bumped this to 4.66 due to needing a higher version of [configuration-as-code-plugin](https://github.com/jenkinsci/configuration-as-code-plugin).
Version [1647.ve39ca_b_829b_42](https://github.com/jenkinsci/configuration-as-code-plugin/releases/tag/1647.ve39ca_b_829b_42) onwards of configuration-as-code includes changes made for the [html 3.x update](https://github.com/jenkinsci/configuration-as-code-plugin/pull/2283). As you can see further down, I have bumped configuration-as-code to the latest release.